user-access-service/build.gradle

```diff
@@ -16,15 +16,27 @@ dependencies {
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
         exclude group: 'com.fasterxml.jackson.module', module: 'jackson-module-parameter-names'
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
+    }
+
 
+    api (group: 'com.amazonaws', name: 'aws-java-sdk-secretsmanager', version: project.ext.awsSdkVersion) {
```

Comment:
api because this library is used by the handlers. the `api` helps us to not replicate the library for each handler